daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Serializability.scala

```diff
@@ -148,6 +148,19 @@ private[validation] object Serializability {
     Env(version, interface, context, SRExceptionArg, tyCon).checkType()
   }
 
+  def checkInterface(
```

Comment:
I pretty much adapted the haskell check in `compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs` to the style of the other functions in this file